Add example of updating an xliff translation file (#685)
amplitude/amplitude-docs: content/collections/guides_and_surveys/en/localization.md

1. Complete the guide or survey's steps in the default language.
2. Download the translation template. This template is a .xliff file that contains [translatable strings](#translatable-strings) from each step of your guide or survey.
    {{partial:admonition type="tip" heading="Adding translations to the xliff files"}}

>     Our template files automatically provide the untranslated content inside `<source>` tags that shouldn't change. Instead, add the translations inside the `<target>` tags, like "Terminer" below:

Reviewer: ```suggestion
    Guides and Surveys template files add untranslated content to `<source>` tags in a .xliff file. The content in these tags shouldn't change. Add translations to the `<target>` tags, like "Terminer" below:
```

Thanks @matthew-beaty!

A few style guide corrections here. But otherwise, LGTM!

>     Our template files automatically provide the orginal untranslated content inside `<source>` tags that should not be edited. Instead, you'll add translations inside the `<target>` tags, like "Terminer" has been added below:

Author: ```suggestion
    Our template files automatically provide the untranslated content inside `<source>` tags that shouldn't change. Instead, add the translations inside the `<target>` tags, like "Terminer" below:
```
